src/nuklear_combo.c

```diff
@@ -778,7 +778,7 @@ nk_combo_string(struct nk_context *ctx, const char *items_separated_by_zeros,
     return nk_combo_separator(ctx, items_separated_by_zeros, '\0', selected, count, item_height, size);
 }
 NK_API int
-nk_combo_callback(struct nk_context *ctx, void(*item_getter)(void*, int, const char**),
+nk_combo_callback(struct nk_context *ctx, void(*item_getter)(void*, int, const char* const *),
```

Comment:
Pretty sure this is a breaking change as old `void (*)(void *, int, const char**)` functions passed here will now be a compiler error.

Reply:
That's right, but that's why there's a major bump and not a minor one. If this change isn't enforced then it won't work for some cases.

Reply:
Bumping to 4.10 is a minor bump. We would need a 5.0 to be a major one.

Reply:
There is actually no major bump (only minor) in this PR as of now :cry:.

But more importantly I'd definitely prefer if major bumping features (i.e. those affecting backwards compatibility) were in a separate PR. It's true that this PR is quite convoluted though I think I'd merge it after it passes our review. But the split due to a major bump is something important for several other reasons (one of which is release engineering).

Reply:
@Hejsil my bad, it's major.minor.bugfix? Then I might have to bump it